src/v3/src/components/Widget/index.tsx

```diff
@@ -13,6 +13,7 @@
 import './style.module.css';
 
 import { ScopedCssBaseline } from '@mui/material';
+import { deepmerge } from '@mui/utils';
```

Comment:
@glenfannin-okta @shawyu-okta any specific reason not to use `@mui/utils`? It performs the same merge as https://gist.github.com/lesterchoi-okta/dbbb928f1b4df03e878e01d5d97c55d2 and is (better) tested.

Reply:
It looks like the deepmerge function provided by MUI also allows an `options` param to avoid cloning the object (for faster processing). Since this is built for MUI theming (and as you mentioned better tested for use in this scenario) I think this is a great replacement.